packages/hardhat-etherscan/test/unit/solc/bytecode.ts

```diff
@@ -79,7 +79,12 @@ describe("Compiler bytecode and deployed bytecode matching", () => {
         moreData: 42,
         mysteriousString: "mystery",
       };
-      const mockMetadata = Buffer.from(encode(mockSolcMetadataMapping));
+      const encoded = encode(mockSolcMetadataMapping);
+      const mockMetadata = Buffer.from(
+        encoded.buffer,
+        encoded.byteOffset,
+        encoded.byteLength
+      );
```

Comment:
I'm curious about this change, what does it mean?

Reply:
Functionally the new code is equivalent to the old code, but I believe this was due to a subtle change in how the `Buffer.from` function types its arguments that caused the object returned from `encode` to not be directly passable to `Buffer.from` anymore. This workaround just utilizes buffer's other two optional arguments that were already provided by `encode` luckily.